openhtf/plugs/user_input.py

```diff
@@ -166,13 +172,12 @@ def respond(self, prompt_id, response):
 
 
 def prompt_for_test_start(
-    message='Provide a DUT ID in order to start the test.', text_input=True,
-    timeout_s=60*60*24):
-  """Make a test start trigger based on prompting the user for input."""
-  def trigger():  # pylint: disable=missing-docstring
-    plug = UserInput()
-    response = plug.prompt(
-        message, text_input=text_input, timeout_s=timeout_s)
-    plug.tearDown()
-    return response
-  return trigger
+    message='Enter a DUT ID in order to start the test.', timeout_s=None):
+  """Return an OpenHTF phase for use as a prompt-based start trigger."""
+  @plugs.plug(prompts=UserInput)
+  def trigger_phase(test, prompts):
```

Comment:
I think you're still going to want to pass a timeout, via normal phase options, otherwise this is going to have the default phase timeout (5 min i think, if not overridden via flag).